exercises/practice/transpose/.meta/example.cairo

```diff
@@ -0,0 +1,35 @@
+pub fn transpose(input: Array<ByteArray>) -> Array<ByteArray> {
+    let mut output: Array<ByteArray> = ArrayTrait::new();
+
+    let mut max_length = 0;
+    for line in input.clone() {
+        if line.len() > max_length {
+            max_length = line.len();
+        }
+    };
+
+    let mut i = 0;
+    loop {
+        let mut temp: ByteArray = "";
+        for line in input
```

Comment:
It must be possible to avoid having a `loop + for + break + continue` combination to perform this operation, makes it very hard to follow what is going on.
 
Find a way to refactor this into something more readable.

You can use [Go implementation](https://github.com/exercism/go/blob/main/exercises/practice/transpose/.meta/example.go) as inspiration, it is very clear and concise